docs/en/user_guides/4_pretrain_custom_dataset.md

```diff
@@ -0,0 +1,370 @@
+  - [Train MAE on COCO Dataset](#train-mae-on-coco-dataset)
+  - [Train SimCLR on Custom Dataset](#train-simclr-on-custom-dataset)
+  - [Load Pre-trained Model to Speedup Convergence](#load-pre-trained-model-to-speedup-convergence)
+
+In this tutorial, we provide some tips on how to conduct self-supervised learning on your own dataset(without the need of label).
+
+## Train MAE on Custom Dataset
+
+In MMSelfSup, We support the `CustomDataset` from MMClassification,  which is able to read the images within the specified folder directly. You only need to prepare the path information of the custom dataset and edit the config.
+
+### Get the path of custom dataset
+
+It should be like `/dataset/my_custom_dataset`
+
+### Choose one config as template
+
+Here, we would like to use `configs/selfsup/mae/mae_vit-base-p16_8xb512-coslr-400e_in1k.py` as the example. We first copy this config file and rename it as `mae_vit-base-p16_8xb512-coslr-400e_custom_dataset.py`.
+
+The content of this config is:
+
+```python
+_base_ = [
+    '../_base_/models/mae_vit-base-p16.py',
+    '../_base_/datasets/imagenet_mae.py',
+    '../_base_/schedules/adamw_coslr-200e_in1k.py',
+    '../_base_/default_runtime.py',
+]
+
+# dataset 8 x 512
+train_dataloader = dict(batch_size=512, num_workers=8)
+
+# optimizer wrapper
+optimizer = dict(
+    type='AdamW', lr=1.5e-4 * 4096 / 256, betas=(0.9, 0.95), weight_decay=0.05)
+optim_wrapper = dict(
+    type='OptimWrapper',
+    optimizer=optimizer,
+    paramwise_cfg=dict(
+        custom_keys={
+            'ln': dict(decay_mult=0.0),
+            'bias': dict(decay_mult=0.0),
+            'pos_embed': dict(decay_mult=0.),
+            'mask_token': dict(decay_mult=0.),
+            'cls_token': dict(decay_mult=0.)
+        }))
+
+# learning rate scheduler
+param_scheduler = [
+    dict(
+        type='LinearLR',
+        start_factor=1e-4,
+        by_epoch=True,
+        begin=0,
+        end=40,
+        convert_to_iter_based=True),
+    dict(
+        type='CosineAnnealingLR',
+        T_max=360,
+        by_epoch=True,
+        begin=40,
+        end=400,
+        convert_to_iter_based=True)
+]
+
+# runtime settings
+# pre-train for 400 epochs
+train_cfg = dict(max_epochs=400)
+default_hooks = dict(
+    logger=dict(type='LoggerHook', interval=100),
+    # only keeps the latest 3 checkpoints
+    checkpoint=dict(type='CheckpointHook', interval=1, max_keep_ckpts=3))
+
+# randomness
+randomness = dict(seed=0, diff_rank_seed=True)
+resume = True
+```
+
+### Edit the dataset related config
+
+The dataset related config is defined in `'../_base_/datasets/imagenet_mae.py'` in `_base_`. We then copy the content of dataset config file into our created file `mae_vit-base-p16_8xb512-coslr-400e_custom_dataset.py`.
+
+- Then we remove the `'../_base_/datasets/imagenet_mae.py'` in `_base_`.
+- Set the `dataset_type = 'mmcls.CustomDataset'`, and the path of the custom dataset ` dataset_type = /dataset/my_custom_dataset`.
```

Comment:
Better to remind the user to add the ``custom_dataset`` in ``mmcls``, instead of ``mmselfsup``.